src/components/Events/EventCard.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+
+import { Row, Col } from '../grid'
+import StyledLink from '../Common/StyledLink'
+
+import { CardTitle, BodyPrimary, CalendarDay } from '../Typography'
+import remcalc from 'remcalc'
+
+const CalendarPage = styled.div`
```

Comment:
Ehm, I was thinking to the old fashioned calendars where you have one day per page and you tear a page off each day - hence CalendarPage [kinda like this one](https://images.app.goo.gl/pMGsfK2c6ayCpjf27)

Reply:
I see, fair point, I would say that considering our context page is usually a reserved word for different pages so this might be a bit confusing.

Reply:
I was tempted by CalendarDate, but then we also have CalendarDay and I think I wanted to avoid confusion between the two. Any suggestions as to what would work best?

Reply:
The naming has gone a bit off here:

We have DataCard which then has several `Calendar*` components within it.
```
const DateCard = ({ date }) => (
  <CalendarPage>
    <CalendarDay noPadding>{date.day}</CalendarDay>
    <CenteredBodyPrimary noPadding>
      {date.month.toUpperCase()}
    </CenteredBodyPrimary>
  </CalendarPage>
)
```

Maybe something like this

```
const DateCard = ({ date }) => (
  <DateWrapper>
    <DateDay noPadding>{date.day}</DateDay>
    <CenteredBodyPrimary noPadding>
      {date.month.toUpperCase()}
    </CenteredBodyPrimary>
  </DateWrapper>
)
```